drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -239,10 +244,10 @@ function DragAndDropTemplates(configuration) {
         });
 
         return (
-            h('section.feedback', {}, [
+            h('div.feedback', {attributes: {'role': 'group', 'aria-label': gettext('Feedback')}}, [
                 h(
                     "div.feedback-content",
-                    { attributes: { 'aria-live': 'polite' } },
+                    {attributes: {'aria-live': 'polite', 'aria-atomic': 'true', 'aria-relevant': 'all'}},
```

Comment:
That's a very good point. It looks like the `aria-live` regions were conflicting with `SR.readText` because after removing `aria-live` attributes, VoiceOver seems to read feedback messages correctly most of the time.